src/wallet/wallet2.cpp

```diff
@@ -1083,7 +1085,8 @@ wallet2::wallet2(network_type nettype, uint64_t kdf_rounds, bool unattended):
   m_unattended(unattended),
   m_devices_registered(false),
   m_device_last_key_image_sync(0),
-  m_use_dns(true)
+  m_use_dns(true),
+  m_export_format(ExportFormat::Raw)
```

Comment:
This does not look like it's saved along with the other settings. If you reload a wallet, it'll be reset.

Reply:
Good catch. Adding.